admin-console/src/main/webapp/src/components/CreateEditModal.tsx

```diff
@@ -0,0 +1,75 @@
+import { ActionGroup, Button, Form, FormGroup, HelperText, HelperTextItem, Modal, ModalVariant,
+    Spinner, TextArea, TextInput } from '@patternfly/react-core';
+import React from 'react';
+
+import { useCreateEventType } from '../services/CreateEventTypes';
+
+interface CreateEditModalProps {
+    isEdit: boolean;
+    showModal: boolean;
+    applicationName?: string;
+    eventTypeName?: string;
+    eventTypeDisplayName?: string;
+    eventTypeDescription?: string;
```

Comment:
Instead of receiving each attribute, we could receive an initializer for the evenType: i.e.:
`initialEventType?: EventType`

and it could be used as:
```
const [ eventType, setEventType ] = React.useState<Partial<EventType>>(props.initalEventType ?? {});
```